package.json

```diff
@@ -48,7 +48,7 @@
     "typescript-eslint-parser": "^21.0.2"
   },
   "dependencies": {
-    "fake-xml-http-request": "^2.0.0",
+    "fake-xml-http-request": "kellyselden/FakeXMLHttpRequest#response",
```

Comment:
It seems this package used in test is from bower.
https://github.com/pretenderjs/pretender/blob/v3.0.4/karma.conf.js#L18

Reply:
Just merged https://github.com/pretenderjs/pretender/pull/281. Can you try rebase?

Reply:
done

Reply:
I think it's github artifact cache didn't update. I just tried v2.1.1 and it passed tests.